src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/IndexProperties.sql

```diff
@@ -0,0 +1,14 @@
+﻿GO
+CREATE TABLE dbo.IndexProperties 
+  (
+     TableName     varchar(100)     NOT NULL
+    ,IndexName     varchar(200)     NOT NULL
+    ,PropertyName  varchar(100)     NOT NULL
+    ,PropertyValue varchar(100)     NOT NULL
+    ,CreateDate    datetime         NOT NULL CONSTRAINT DF_IndexProperties_CreateDate DEFAULT getUTCdate()
+    
+     CONSTRAINT PKC_IndexProperties_TableName_IndexName_PropertyName PRIMARY KEY CLUSTERED (TableName, IndexName, PropertyName)
+  )
+GO
+--INSERT INTO IndexProperties (TableName,IndexName,PropertyName,PropertyValue) 
+--  SELECT 'ReferenceSearchParam', 'IX_ReferenceSearchParam_SearchParamId_ReferenceResourceTypeId_ReferenceResourceId_BaseUri_ReferenceResourceVersion', 'DATA_COMPRESSION', 'PAGE'
```

Comment:
Do we need this if commented out?

Reply:
Technically we do not need this but it is valuable for debugging. I suggest to leave as is.